openassessment/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
 Initialization Information for Open Assessment Module
 """
-__version__ = '5.0.4'
+__version__ = '5.0.5'
```

Comment:
nit: [`master`](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) is currently at `'5.1.0'`, so:
```suggestion
__version__ = '5.1.1'
```